actioncable/lib/action_cable.rb

```diff
@@ -32,7 +32,8 @@ module ActionCable
     message_types: {
       welcome: 'welcome'.freeze,
       ping: 'ping'.freeze,
-      confirmation: 'confirm_subscription'.freeze,
+      confirm_subscribe: 'confirm_subscribe'.freeze,
```

Comment:
Will this break existing clients when the server is updated? Or did the Action Cable protocol stuff fix that incompatibility?